### PR TITLE
HW7

### PR DESCRIPTION
**Pull request by Oksana-robota**

### CODE REVIEW

File: HW7/Homework_7.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+while True:
+    commands = input('Write command:\n')
+    if commands == 'stats':
+        print(f'Amount of records is {len(phone_book)}')
+    elif commands == 'list':
+        print(*phone_book.keys(), sep='\n')
+    elif commands in ['delete', 'show']:
+        name = input('Write name:\n')
+        if commands == 'delete':
+            del phone_book[name]
```
Comment (klymenok) [inline HW7/Homework_7.py:11]: It is better to check if key exists before attempt to delete it

File: HW7/Homework_7.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+            print(f'Record with name "{name}" was successfully deleted')
+        elif commands == 'show':
+            print(f'Here you could see the info for "{name}": {phone_book[name]}')
```
Comment (klymenok) [inline HW7/Homework_7.py:14]: It is better to check if key exists before attempt to access it

**Review by klymenok — CHANGES_REQUESTED** [review]

**Comment by Oksana-robota** [comment]
Could you please check the homework for the 9th lecture. I'm not sure about the second task. So that, I provided to options (a and b). Option a I did on my own, the way I was thinking and my understanding of the iterators. Option b I googled classes for better understanding of them and did it using classes' examples. I would be glad if you comment the second task anyway. Thank you!

File: HW9/Homework_9.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
+# 1. Fibonacci sequences using generator
```
Comment (klymenok) [inline HW9/Homework_9.py]: This code shouldn't be here

**Review by klymenok — CHANGES_REQUESTED** [review]

**Review by klymenok — APPROVED** [review]
Looks good!

**Pull request merged**